packages/block-library/src/missing/style.native.scss

```diff
@@ -1,7 +1,72 @@
 /** @format */
+.content {
+	padding-top: 8;
+	padding-bottom: 0;
+	padding-left: 24;
+	padding-right: 24;
+	align-items: center;
+	justify-content: space-evenly;
+}
+
+.helpIconContainer {
+	position: absolute;
+	top: 0;
+	right: 0;
+	height: 48;
+	width: 48;
+	padding-top: 12;
+	padding-right: 12;
+	justify-content: flex-start;
+	align-items: flex-end;
+}
+
+.infoContainer {
+	flex-direction: column;
+	align-items: center;
+	justify-content: flex-end;
+}
+
+.infoIcon {
+	size: 36;
+	height: 36;
+	padding-top: 8;
+	padding-bottom: 8;
+}
+
+.infoText {
+	text-align: center;
+}
+
+.infoTitle {
+	padding-top: 8;
+	padding-bottom: 12;
+	font-size: 20;
+	font-weight: bold;
+	color: $gray-dark;
+}
+
+.infoDescription {
```

Comment:
Same as for `infoTitle`, we should create a `infoDescriptionDark` class and update the color.

Reply:
Addressed with 5a9e429.